snapm/manager/_mounts.py

```
@@ -0,0 +1,682 @@
+    These must be passed to avoid destroying any existing quota information:
+
+        https://github.com/lvmteam/lvm2/issues/182
+        https://github.com/storaged-project/libblockdev/pull/1132
+
+    :param devpath: The path to the block device to examine.
+    :returns: A comma-separated string list of mount options.
+    :rtype: `str`
+    """
+    xfs_db_cmd = ["xfs_db", "-r", devpath, "-c", "sb 0", "-c", "print qflags"]
+    try:
+        result = run(
```

Ruff (0.14.1)

`subprocess` call: check for execution of untrusted input

(S603)